flutter-studio/src/io/flutter/project/FlutterProjectStep.java

```diff
@@ -279,6 +297,7 @@ protected void onEntering() {
     }
     myHeading.setText(heading);
     myDownloadErrorMessage.set("");
+    mySdkPathContent = (String)myFlutterSdkPath.getComboBox().getEditor().getItem();
```

Comment:
just checking that `myFlutterSdkPath.getComboBox()` can't be generified in some way (to avoid the cast)?

Reply:
Not that I know of. ComboboxWithBrowseButton doesn't define any type variables.